src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -266,6 +268,13 @@ void ThreadStateBar::DoUpdatePrimitives(PrimitiveAssembler& primitive_assembler,
 void ThreadStateBar::OnPick(int x, int y) {
   ThreadBar::OnPick(x, y);
   app_->set_selected_thread_id(GetThreadId());
+
+  Vec2i screen_cords = Vec2i(x, y);
+  float clicked_x = viewport_->ScreenToWorld(screen_cords)[0];
```

Comment:
`clicked_world_x`, the coordinate system is a mess, better if you called it world